quantum/nodes/simulator/simulator.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const util = require('util');
+const snippets = require('../../snippets');
+const shell = require('../../python').PythonShell;
+const {errors} = require('../../errors');
+
+module.exports = function(RED) {
+  const validateInput = (node, msg) => {
+    if (msg.topic !== 'Quantum Circuit') {
+      node.error(errors.NOT_QUANTUM_CIRCUIT, msg);
+    }
+  };
+  function SimulatorNode(config) {
```

Comment:
```suggestion
const util = require('util');
const snippets = require('../../snippets');
const shell = require('../../python').PythonShell;
const {errors} = require('../../errors');

const validateInput = (node, msg) => {
  if (msg.topic !== 'Quantum Circuit') {
    node.error(errors.NOT_QUANTUM_CIRCUIT, msg);
  }
};

module.exports = function(RED) {
  function SimulatorNode(config) {
```
I think it would be clearer to have this function stored at the top-level.